### PR TITLE
Resolve broken links to Insider Preview builds

### LINKED ISSUE #9241: Broken link
The link is broken:

**Reset this PC**
The 'reset this PC' recovery function now includes a *cloud download option*.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d651de3c-b8eb-e8d9-8138-1eadb758bf2e
* Version Independent ID: a19a8b65-0f4a-efd5-7f83-56bd07cc8c3b
* Content: [What's new in Windows 10, version 2004 - What's new in Windows](https://docs.microsoft.com/en-us/windows/whats-new/whats-new-windows-10-version-2004)
* Content Source: [windows/whats-new/whats-new-windows-10-version-2004.md](https://github.com/MicrosoftDocs/windows-itpro-docs/blob/public/windows/whats-new/whats-new-windows-10-version-2004.md)
* Product: **w10**
* Technology: **windows**
* GitHub Login: @greg-lindsay
* Microsoft Alias: **greglin**

### PR DESCRIPTION
**Pull request by illfated**
**Resolve broken links in whats-new-windows-10-version-2004.md**

From issue ticket #9241:

> The link is broken:
>
> **Reset this PC**
> The 'reset this PC' recovery function now includes a cloud download option.

Thanks to @CoMMyCZ for noticing & reporting this issue.

**Changes proposed:**

- Replace 8 occurrences of the outdated `/at-work-pro/wip-4-biz-whats-new` directory & page links

**Codestyle & whitespace:**

- Remove redundant leading space before dash in the "See Also" bullet point list
- Remove redundant HTML `<br>` tags from "See Also" sentence endings (handled by MarkDown)
- Remove any redundant end-of-line (EOL) whitespace (blanks)

Closes #9241

### CODE REVIEW

**Comment by illfated** [comment]
@JohanFreelancer9 : Kindly perform a copy review for this PR. Please note that a couple of the link updates also contain preliminary suggestions for updated text by copying the section heading from the Archive page, making sentence structure changes. I will need suggestions for better phrasing to make sure that MS Docs codestyle is preserved.

**Review by illfated — COMMENTED** [review]
**Hide whitespace changes** (removes a couple lines where the change is only a cleanup of end-of-line blanks):
- https://github.com/MicrosoftDocs/windows-itpro-docs/pull/9244/files?diff=unified&w=1

**Rich Diff view** for readability review: https://github.com/MicrosoftDocs/windows-itpro-docs/pull/9244/files?diff=unified&w=1&short_path=4d09d59#diff-4d09d5969635a11bf502b5c5cb469f854d493679cc35b0b0b08b9f31d3ffb365

**Comment by illfated** [comment]
#sign-off

**Comment by nam31** [comment]
@JohanFreelancer9 please do the copy editor review for this PR.
Thank you

File: windows/whats-new/whats-new-windows-10-version-2004.md [hunk 1]
```diff
@@ -108,15 +108,15 @@ Windows PowerShell cmdlets have been improved:
...
 
 Additional improvements:
-- Enterprise network [throttling is enhanced](https://docs.microsoft.com/windows-insider/at-work-pro/wip-4-biz-whats-new#new-download-throttling-options-for-delivery-optimization-build-18917) to optimize foreground vs. background throttling.
+- Enterprise network [throttling is enhanced](https://docs.microsoft.com/windows-insider/archive/new-in-20H1#new-download-throttling-options-for-delivery-optimization-build-18917) to optimize foreground vs. background throttling.
 - Automatic cloud-based congestion detection is available for PCs with cloud service support.
 
 The following [Delivery Optimization](https://docs.microsoft.com/windows/deployment/update/waas-delivery-optimization) policies are removed in this release:
 
 - Percentage of Maximum Download Bandwidth (DOPercentageMaxDownloadBandwidth)
   - Reason: Replaced with separate policies for foreground and background
```
Comment (JohanFreelancer9) [inline windows/whats-new/whats-new-windows-10-version-2004.md:117]:
```suggestion
  - Reason: Replaced with separate policies for foreground and background.
```

File: windows/whats-new/whats-new-windows-10-version-2004.md [hunk 2]
```diff
@@ -108,15 +108,15 @@ Windows PowerShell cmdlets have been improved:
...
 - Max Upload Bandwidth (DOMaxUploadBandwidth)
-  - Reason: impacts uploads to internet peers only, which isn't used in Enterprises. 
+  - Reason: impacts uploads to internet peers only, which isn't used in Enterprises.
```
Comment (JohanFreelancer9) [inline windows/whats-new/whats-new-windows-10-version-2004.md:119]:
```suggestion
  - Reason: Impacts uploads to internet peers only, which isn't used in enterprises.
```

File: windows/whats-new/whats-new-windows-10-version-2004.md [hunk 3]
```diff
@@ -108,15 +108,15 @@ Windows PowerShell cmdlets have been improved:
...
 - Absolute max throttle (DOMaxDownloadBandwidth)
   - Reason: separated to foreground and background
```
Comment (JohanFreelancer9) [inline windows/whats-new/whats-new-windows-10-version-2004.md:121]:
```suggestion
  - Reason: Separated to foreground and background.
```

File: windows/whats-new/whats-new-windows-10-version-2004.md [hunk 4]
```diff
@@ -232,7 +232,7 @@ The following items are added to Task Manager in this release:
 
 ### 2-in-1 PCs
 
-A [new tablet experience](https://docs.microsoft.com/windows-insider/at-work-pro/wip-4-biz-whats-new#new-tablet-experience-for-2-in-1-convertible-pcs-build-18970) for two-in-one convertible PCs is available. The screen will be optimized for touch when you detach your two-in-one's keyboard, but you'll still keep the familiar look of your desktop without interruption. 
+[Introducing a new tablet experience for 2-in-1 convertible PCs! (Build 18970)](https://docs.microsoft.com/windows-insider/archive/new-in-20H1#introducing-a-new-tablet-experience-for-2-in-1-convertible-pcs-build-18970) A new tablet experience for two-in-one convertible PCs is available. The screen will be optimized for touch when you detach your two-in-one's keyboard, but you'll still keep the familiar look of your desktop without interruption.
```
Comment (JohanFreelancer9) [inline windows/whats-new/whats-new-windows-10-version-2004.md:235]:
```suggestion
See [Introducing a new tablet experience for 2-in-1 convertible PCs! (Build 18970)](https://docs.microsoft.com/windows-insider/archive/new-in-20H1#introducing-a-new-tablet-experience-for-2-in-1-convertible-pcs-build-18970) for details on a new tablet experience for two-in-one convertible PCs that is now available. The screen will be optimized for touch when you detach your two-in-one's keyboard, but you'll still keep the familiar look of your desktop without interruption.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - looks good, just a few minor suggestions.

**Comment by illfated** [comment]
Very good, thank you for the added corrections. Appreciated.

**Comment by illfated** [comment]
#sign-off

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by illfated** [comment]
Copy edit review done. PR ready for final review & merge.

**Comment by nam31** [comment]
@greg-lindsay PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**